server_admin/topics/authentication/webauthn.adoc

```diff
@@ -244,3 +244,52 @@ The final configuration of the flow looks similar to this:
 image:images/webauthn-passwordless-flow.png[]
 
 You can now add *WebAuthn Register Passwordless* as the required action to a user, already known to {project_name}, to test this. During the first authentication, the user must use the password and second-factor WebAuthn credential. The user does not need to provide the password and second-factor WebAuthn credential if they use the WebAuthn Passwordless credential.
+
+[[_webauthn_loginless]]
+==== LoginLess WebAuthn
+
+{project_name} uses WebAuthn for two-factor authentication, but you can use WebAuthn as the first-factor authentication. In this case, users with `passwordless` WebAuthn credentials can authenticate to {project_name} without submitting a login nor a password. {project_name} can use WebAuthn as both the loginless/passwordless and two-factor authentication mechanism in the context of a realm.
+
+An administrator typically requires that Security Keys registered by users for the WebAuthn loginless authentication meet different requirements. Loginless authentication requires users to authenticate to the security key (eg. by using a PIN code or a fingerprint) and that the cryptographic keys associated with the loginless credential are stored physically on the securitey key (not all security keys meet that kind of requirements, please check with your security key vendor if your device supports 'user verification' and 'resident key', see <<_webauthn-supported-keys, Supported Security Keys>>)
```

Comment:
change "eg" to "for example
Correct spelling of securitey
Remove the parenthesis and change the parenthetical content into three sentences beginning "Not," "Please," and "See."